apps/examples/testcase/Kconfig

```diff
@@ -1,49 +0,0 @@
-#
-# For a description of the syntax of this configuration file,
-# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
-#
-
-menuconfig EXAMPLES_TESTCASE
-	bool "TestCase Example"
-	default n
-	---help---
-		Enable the TestCase Example
-
-if EXAMPLES_TESTCASE
-config DISABLE_MANUAL_TESTCASE
```

Comment:
This is a necessary config. Could you add it at somewhere?

Reply:
The right place to add it seems to be this folder's Kconfig only,
So, changes to this folder have been reverted